homeassistant/components/insteon/manifest.json

```diff
@@ -2,8 +2,12 @@
   "domain": "insteon",
   "name": "Insteon",
   "documentation": "https://www.home-assistant.io/integrations/insteon",
-  "requirements": ["pyinsteon==1.0.9"],
-  "codeowners": ["@teharris1"],
+  "requirements": [
+    "pyinsteon==1.0.11"
```

Comment:
The fix for for python 3.9 breakage is included in 1.0.12 not 1.0.11

Reply:
This is not related to this PR. The code-owner has updated it to 1.0.11, if it needs to be bumped to 1.0.12, that is possible, but would need another PR towards the Core codebase.

Based on its contents we can evaluate if that can be included in the next Home Assistant patch release.

At this point, it simply does not exist at our end, so we cannot release what you suggest.

Reply:
3 hours ago we got a PR that said 11 fixed Python 3.9 compat https://github.com/home-assistant/core/pull/52927

Reply:
Ah I see it now. That is indeed not correct. We will wait for someone to bump and verify the 12 release works. It won't make this hotfix because I don't have insteon here to test a bump.